web/src/app/projects/view/view.component.ts

```diff
@@ -40,9 +40,9 @@ export class ViewProjectComponent implements OnInit {
     }
 
     ngOnInit(): void {
-        this.projectSubscription = this.projectService.findOneById(this.project.uid).subscribe((project: ProjectModel) => {
-                this.project = project;
-            });
+        this.projectSubscription = this.projectService
+            .findOneById(this.project.uid)
+            .subscribe((project: ProjectModel) => { this.project = project; });
```

Comment:
the curly brackets are not needed 😄 

https://github.com/DashboardHub/PipelineDashboard/pull/1194#discussion_r279042504

Reply:
Point to be noted !! Fixed

Reply:
👍 🤓